tests/app/celery/test_nightly_tasks.py

```diff
@@ -316,6 +325,10 @@ def test_delete_notifications_task_calls_task_for_services_that_have_sent_notifi
     ])
 
 
+def delete_notifications_by_service_and_type(id, param, param1):
+    pass
+
+
```

Comment:
Does this test still need to be fully fleshed out?

Reply:
oops, will fix.